### ISSUE TITLE
Clarify what expiration date means in UI

### ISSUE THREAD

**Issue by adrake33** [AugurProject/augur-app#419]
Joey posted on Discord:
```
Also UI has lack of info on expiration date and what it means in market creation
The UI should say that expiration date is sometime after when the result is known in market creation trading page and reporting
Users may think that it means trading will stop then or any other stuff
```

Currently, the Create Market UI has a tooltip saying:
```
Market Expiration Date

Augur is still experimental software, creating markets past six months has been disabled to discourage long running markets.
```

However, the trading page and market cards on the market list page do not have tooltips explaining the expiration date. Also, reporting/dispute pages & market cards on the reporting pages do not have tooltips for the expiration date.

Suggest modifying the Create Market copy to be:
```
Market Expiration Date

The expiration date is when the result of the market should be known. Trading may still occur after this time.

Augur is still experimental software, creating markets past six months has been disabled to discourage long running markets.
```

and adding tooltips to the market cards, trading page, & reporting/dispute pages with the following copy:
```
Market Expiration Date

The expiration date is when the result of the market should be known. Trading may still occur after this time.
```

**Comment by adrake33** [comment]
Do the above changes sound ok, @sharkcrayon @matt-bullock ?

**Comment by 0xpunk** [comment]
Please also add somewhere the correct timezone. See: https://github.com/AugurProject/augur-ui/issues/2409 (dunno why it was closed with no solution, but if expiration date is after some daylight savings time event, then the creation form and the actual submitted market will use two different timezones, with no indication that something is wrong until it's too late)

It would actually be nice to see the final UTC timestamp somewhere before submission, like maybe on the final confirmation screen. It would help not only with this DST issue but also just simple user error, like when commonly making a market to expire at midnight UTC it would be better to see that and not just local time.

**Comment by damian-c** [comment]
From comments I've seen repeatedly on Reddit, it appears people are confused by the term "expiration date", often assuming that it means trading ends at this time. Your suggestions are good, but not everyone will read the tool tips. Perhaps an alternative would be just replacing all occurrences of "expiration date" in the UI with something like "reporting begins"?

**Comment by bthaile** [comment]
Expiration date is confusing and we’ve had a few discussions about a better term. 

Its not intuitive that augur markets are tradable forever. So nothing expires, but reporting begins, would need explaining as well. 

More suggesting are welcomed

**Comment by veox** [comment]
This issue seems to have been addressed in v1.11.0.

**Comment by bthaile** [comment]
closing, there are existing tix to change End Time to `Event Ends` and `Reporting Start Time`
